cli/main.rs

```diff
@@ -315,12 +314,8 @@ fn init_env() {
 async fn main() {
     init_env();
 
-    use sc_executor::{sp_wasm_interface::ExtendedHostFunctions, NativeExecutionDispatch};
     let cmd = TryRuntime::parse();
-    cmd.run::<Block, ExtendedHostFunctions<
-        sp_io::SubstrateHostFunctions,
-        <ExecutorDispatch as NativeExecutionDispatch>::ExtendHostFunctions,
-    >>()
-    .await
-    .unwrap();
+    cmd.run::<Block, sp_io::SubstrateHostFunctions>()
```

Comment:
Is there an easy way to mock some of these host functions? With a macro maybe?  
The last time i tried i had to manually implement them on a mocked struct, but it looked a bit ugly since i needed thread locals to configure because of no `self` pointer.  
see https://github.com/paritytech/try-runtime-cli/issues/56